src/app/portlets/dot-edit-page/content/components/dot-form-selector/dot-form-selector.component.ts

```diff
@@ -39,8 +56,17 @@ export class DotFormSelectorComponent implements OnInit {
                 this.messages = messages;
             });
 
+        this.paginatorService.paginationPerPage = 5;
         this.paginatorService.url = 'v1/contenttype?type=FORM';
     }
+    ngOnChanges(changes: SimpleChanges) {
+        setTimeout(() => {
+            if (changes.show.currentValue === true) {
+                this.contentMinHeight =
+                    this.dialog.dialog.nativeElement.getBoundingClientRect().height - 100;
+            }
+        }, 0);
+    }
 
     /**
      * Call when click on any pagination link
```

Comment:
test?

Reply:
this is the component that has the tests `disabled`
https://github.com/dotCMS/core-web/blob/448b543f7ed45db45585b097689a67be111ef3e6/src/app/portlets/dot-edit-page/content/components/dot-form-selector/dot-form-selector.component.spec.ts

for some reason when you enable it, fails completely